EIPS/eip-2364.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: 2364
+title: eth/64: forkid-extended protocol handshake
```

Comment:
```suggestion
title: "eth/64: forkid-extended protocol handshake"
```

Need to be enclosed in quotes otherwise the yaml parser considers it structure. During display the quotes are excluded (the hardfork metas do the same thing).